src/Sylius/Bundle/ApiBundle/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -595,7 +595,7 @@ private function isAllowedProperty(string $class, string $property): bool
         try {
             $reflectionProperty = new \ReflectionProperty($class, $property);
 
-            return $reflectionProperty->getModifiers() & $this->propertyReflectionFlags;
+            return boolval($reflectionProperty->getModifiers() & $this->propertyReflectionFlags);
```

Comment:
- 
```suggestion
            return (bool) ($reflectionProperty->getModifiers() & $this->propertyReflectionFlags);
```

Reply:
It not work with this case:
<img width="1102" alt="Screenshot 2021-05-04 at 11 27 20" src="https://user-images.githubusercontent.com/39232096/116984789-e8435f00-accb-11eb-8039-319141d7e439.png">

I try to add this fix in symfony and using `boolval` working properly, but casting with `(bool)` throw many errors: 
<img width="1148" alt="Screenshot 2021-05-04 at 11 26 09" src="https://user-images.githubusercontent.com/39232096/116984916-17f26700-accc-11eb-91b6-bfa79ef00d0d.png">